### PR TITLE
SB-18378

### PR DESCRIPTION
**Pull request by PradyumnaNagendra**

### CODE REVIEW

**Comment by rjshrjndrn** [comment]
@SMY-ALTAMASH please have a look

File: ansible/inventory/env/group_vars/all.yml [hunk 1]
```diff
@@ -197,6 +197,7 @@ sunbird_dial_repo_api_key: "{{ core_vault_sunbird_ekstep_api_key }}"
 sunbird_plugin_repo_api_base_url: "http://{{searchservice_ip}}:9000" 
 sunbird_data_service_api_base_url: "{{sunbird_ekstep_api_base_url}}"
 sunbird_data_service_api_key: "{{ core_vault_sunbird_ekstep_api_key }}"
+sunbird_content_service_api_base_url: "http://{{sunbird_swarm_manager_lb_ip}}:9002"
```
Comment (rjshrjndrn) [inline ansible/inventory/env/group_vars/all.yml]: is it correct @G33tha

Comment (G33tha) [reply ansible/inventory/env/group_vars/all.yml]: @rjshrjndrn  its correct, we have to override this variable in private repo common.yml to point it to private ingress loadbalancer ip.
@PradyumnaNagendra please update this variable in private repo also, 
 http://{{private_ingressgateway_ip}}/<service>

Comment (rjshrjndrn) [reply ansible/inventory/env/group_vars/all.yml]: :+1:

Comment (PradyumnaNagendra) [reply ansible/inventory/env/group_vars/all.yml]: ok

**Pull request merged**